### DISCUSSION TITLE
Can not paste string to citrix

### DISCUSSION THREAD

**Discussion by guiqiusa** [bbidulock/icewm#661] [Q&A]
I have a citrix vdi with linux operation. This linux use icewm as Xwindow.
I login my win10 notebook,and start my vdi .
can not copy string from win10 to icewm ,but it can copy from win10 to kde or gnome.
Can you give some ideas? some wrong I do it ?

linux os information:
CentOS Linux release 7.8.2003 (Core)
3.10.0-1127.el7.x86_64 #1 SMP Tue Mar 31 23:36:51 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

Ctirix Verison: 1912LTS

icewm version: IceWM 2.9.9.3, Copyright 1997-2012 Marko Macek, 2001 Mathias Hasselmann. 

windows version:
Windows 10  pro , 19044.1889

**Comment by gijsbers** [comment]
https://stackoverflow.com/questions/40213996/vmware-12-cant-copy-paste-between-host-and-virtual-computer
